### PR TITLE
fix: position of browse and create buttons #1424

### PR DESCRIPTION
**Pull request by clevercoderjoy**
# Description

Fixed the overlapping create and browse buttons in the footer section for all small and medium-sized devices.

Fixes https://github.com/reactplay/react-play/issues/1424

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

# Screenshots or example output

image
image

### CODE REVIEW

File: src/common/footer/ExtendedFooter.jsx [hunk 1]
```diff
@@ -39,10 +36,10 @@ const ExtendedFooter = () => {
...
     },
     {
-      name: 'Meetup',
-      url: 'https://www.meetup.com/reactplay-bengaluru/',
-      icon: <FaMeetup className="icon" />,
-      title: 'Meetup Page'
+      name: 'Email',
+      url: 'mailto:contact@reactplay.io',
+      icon: <FaEnvelope className="icon" />,
+      title: 'Email'
```
Comment (priyankarpal) [inline src/common/footer/ExtendedFooter.jsx:42]: please try to add an another object for `email` we added a `meetup` link

**Review by priyankarpal — CHANGES_REQUESTED** [review]
also there are few formatting issue please fix

File: src/common/footer/ExtendedFooter.jsx (hunk 1 above)
Comment (clevercoderjoy) [reply src/common/footer/ExtendedFooter.jsx:42]: done

**Review by priyankarpal — APPROVED** [review]

File: src/common/footer/ExtendedFooter.jsx
Comment (priyankarpal) [inline src/common/footer/ExtendedFooter.jsx:1]: hello @clevercoderjoy but the email is missing. please add the email too

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: src/common/footer/ExtendedFooter.jsx
Comment (clevercoderjoy) [reply src/common/footer/ExtendedFooter.jsx:1]: added

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉 I also suggest keeping branch names short and clear.
example `fix-issue-1414`

**Pull request merged**